build.gradle.kts

```diff
@@ -49,7 +50,13 @@ tasks.register<Exec>("buildDockerImage") {
     if (project.properties["onlyBuild"].toString().toBoolean()) {
         commandLine("docker", "build", ".")
     } else {
-        commandLine("docker", "build", ".", "--tag", "$tempDockerTag")
+        commandLine("docker", "build",
+            "--build-arg", "SPRING_PROFILE=${System.getenv()["SPRING_PROFILE"]}",
+            "--build-arg", "GOOGLE_CLOUD_SQL_POSTGRES_USER=${System.getenv()["GOOGLE_CLOUD_SQL_POSTGRES_USER"]}",
+            "--build-arg", "GOOGLE_CLOUD_SQL_POSTGRES_PASSWORD=${System.getenv()["GOOGLE_CLOUD_SQL_POSTGRES_PASSWORD"]}",
+            "--build-arg", "ION_CORE_BASE_URL=${System.getenv()["ION_CORE_BASE_URL"]}",
+            "--build-arg", "ION_CORE_TOKEN=${System.getenv()["ION_CORE_TOKEN"]}",
```

Comment:
This means we are creating docker images with secrets and environment specific information inside, right?
I don't like this approach. Would prefer images that are independent of the environment and don't have any secrets.

Reply:
How would we configure these properties that are specific to each environment (sta or prod)?

Reply:
*if possible* they should be provided by the environment and not by the application image, which _preferably_ should be environment independent.
If that is not possible due to our deployment model, I was wondering if we could instead create _app images_ (environment independent) and _environment images_ (on top of the _app images_), with the environment information

Reply:
The task `buildDockerImage` if called with `-PonlyBuild=true` just builds the docker image without any argument